ClubMembersSortedCF/src/members/v1/zipCodeValidatorTest.java

```diff
@@ -0,0 +1,22 @@
+package members.v1;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class zipCodeValidatorTest {
+    @ParameterizedTest
+    @ValueSource(ints = {999, 0123, 12345,})
+    void zipCodeValidatorTestA(int number){
+        ZipCodeValidator zipCodeValidator = new ZipCodeValidator();
```

Comment:
please check the `@BeforeAll` and `@BeforeEach` annotations and refactor this part